RNA-seq/00b-additional_approaches.md

```diff
@@ -0,0 +1,135 @@
+# Additional bulk RNA-seq tools and approaches
+
+Our modules are designed to take up approximately half a day and to be run on your own laptops.
+As a result, we do not cover every tool or method that would complement what is included in our main bulk RNA-seq module.
+For your reference, we've put together what we might teach if we had more time or computational resources.
+It is intended to be an introduction to these approaches and a "jumping off point" for further reading or experimentation.
+We present them in order of increasing difficulty and/or departure from what is presented in training.
+ 
+####  Contents
+
+* [MultiQC](#multiqc)
+* [tximeta](#tximeta)
+* [Decoy sequence-aware selective alignment with Salmon](#decoy-sequence-aware-selective-alignment-with-salmon)
+
+## MultiQC
+
+[MultiQC](https://multiqc.info/) is a tool that aggregates output from many tools (73 are currently supported; see the list [here](https://multiqc.info/docs/#multiqc-modules)) into a single HTML report ([Ewels et al. _Bioinformatics._ 2016.](http://dx.doi.org/10.1093/bioinformatics/btw354)).
```

Comment:
Not exactly. It supports Salmon, which is categorized as an alignment tool by MultiQC. So, for instance, it will report Salmon's mapping rate which is arguably a quality control measure but not from a QC tool.